addons/web/static/src/search/search_panel/search_view.scss

```diff
@@ -132,13 +124,17 @@
 
         .o_generator_menu_delete {
             @include o-hover-opacity(0.8, 1);
-            @include o-position-absolute($dropdown-item-padding-x*.3, $dropdown-item-padding-x*.2, auto, auto);
             
             &:hover {
                 background-color: map-get($grays, '100');
             }
         }
     }
+
+    select, input {
```

Comment:
I'd prefer to don't target tags directly, there are classes you can rely on I guess.